src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -2,30 +2,60 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonSetter;
 import com.google.common.collect.Lists;
 
 public class EventType {
```

Comment:
@v-stepanov you're right again. I completely missed this bean setup. I was even doing some local setup that should be global. I made some changes here https://github.com/zalando/nakadi/commit/db91dfce4e2e53c2af45caff9eae54481586d57f